src/features/exporter/Downloader.js

```diff
@@ -96,7 +100,11 @@ class Downloader extends Component {
     let link = document.createElement('a')
     link.download = fileName
 
-    let blob = new Blob([text],{type: 'text/plain;charset=utf-8'})
+    let fileType = 'text/plain;charset=utf-8'
+    if (this.props.fileType === 'SVG (.svg)') {
```

Comment:
I don't like using this key string over and over. I would also like to easily change it, since it is user facing.